projects/kagla/index.js

```diff
@@ -0,0 +1,19 @@
+const utils = require("../helper/utils");
+
+async function astar() {
+  let staked = await utils.fetchURL("https://api.kagla.finance/api/kagla/tvl"); //stable pools
+  return staked.data.tvl;
+}
+
+function fetch() {
+  return astar();
+}
+
+module.exports = {
+  timetravel: false,
+  misrepresentedTokens: true,
+  astar: {
+    fetch: astar,
```

Comment:
Can this be calculated using on-chain calls instead? We've got a guide on how to use our SDK here: https://docs.llama.fi/list-your-project/submit-a-project

Reply:
Sure. I'll try it later!